pykechain/defaults.py

```diff
@@ -99,6 +99,12 @@
     'property2': {'fields': ",".join(
         ['id', 'name', 'ref', 'created_at', 'updated_at', 'model_id', 'part_id', 'order', 'scope_id', 'category',
          'property_type', 'value', 'value_options', 'output', 'description', 'unit'])},
+    'service':  {'fields': ",".join(
```

Comment:
sim does not have this 'fields' info needed in the request. It will work, but is optional in any case.